modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,185 @@
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration similiar to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker <2>
+# ...
+    interfaces:
+    - name: enp2s0 <3>
+      type: ethernet <4>
+      state: up <5>
+      ipv4:
+        enabled: false <6>
+      ipv6:
+        enabled: false
+    - name: br-ex
+      type: ovs-bridge
+      state: up
+      ipv4:
+         enabled: false
+         dhcp: false
+      ipv6:
+         enabled: false
+         dhcp: false
+      bridge:
+       port:
+       - name: enp2s0 <7>
+       - name: br-ex
+    - name: br-ex
+      type: ovs-interface
+      state: up
+      copy-mac-from: enp2s0
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: false
+        dhcp: false
+# ...
+----
+<1> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. If you have a single global configuration specified in an `/etc/nmstate/openshift/cluster.yml` configuration file that you want to apply to all nodes in your cluster, you do not need to specify the hostname path for each node. The `worker` role is the default role for nodes in your cluster. Note that the `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
+<2> The name of the policy.
+<3> Name of the interface.
+<4> The type of interface. This example creates a bridge.
+<5> The requested state for the interface after creation.
+<6> Disables IPv4 and IPv6 in this example.
+<7> The node NIC to which the bridge attaches.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
```

Comment:
We still need to make it clear that this is a day 2 activity, not something you do as a normal part of your day 0/1 installation process.

Reply:
OK, but this section will only show in the Day-2 docs, so do I still need to call it out?